massa-ledger-worker/src/ledger_db.rs

```diff
@@ -166,6 +167,13 @@ impl LedgerDB {
             amount_serializer: AmountSerializer::new(),
             bytecode_serializer: BytecodeSerializer::new(),
             slot_serializer: SlotSerializer::new(),
+            slot_deserializer: SlotDeserializer::new(
+                (Bound::Included(u64::MIN), Bound::Included(u64::MAX)),
+                (
+                    Bound::Included(u8::MIN),
+                    Bound::Included(thread_count.saturating_sub(1)),
+                ),
+            ),
```

Comment:
Consider using included/excluded without the saturating sub (similar idea to using `0..thread_count` instead of `0..=(thread_count - 1)`

Reply:
Makes sense, thanks! Done in 850eac71d0b6228262e323cfcf595109ec266ca8

Reply:
nice. 

Come to think of it, try`0` instead of `u8::MIN`?

Once you've given it thought, go with whichever you think is better, then go ahead and resolve-mark this thread